aspnetcore/data/ef-mvc/migrations.md

```diff
@@ -95,15 +95,13 @@ This code is for the initial migration that was created when you entered the `mi
 
 If you created the initial migration when the database already exists, the database creation code is generated but it doesn't have to run because the database already matches the data model. When you deploy the app to another environment where the database doesn't exist yet, this code will run to create your database, so it's a good idea to test it first. That's why you changed the name of the database in the connection string earlier -- so that migrations can create a new one from scratch.
 
-## Examine the data model snapshot
+## The data model snapshot
 
-Migrations also creates a *snapshot* of the current database schema in *Migrations/SchoolContextModelSnapshot.cs*. Here's what that code looks like:
+Migrations creates a *snapshot* of the current database schema in *Migrations/SchoolContextModelSnapshot.cs*. 
 
-[!code-csharp[](intro/samples/cu/Migrations/SchoolContextModelSnapshot1.cs?name=snippet_Truncate)]
+To delete a migration, use the [dotnet ef migrations remove](https://docs.microsoft.com/ef/core/miscellaneous/cli/dotnet#dotnet-ef-migrations-remove) command.
```

Comment:
Without the sentence that begins "The snapshot file has to be kept in sync..." to introduce this sentence, this sentence doesn't make a lot of sense here because it won't be clear why we're talking about deleting a migration in a section titled "The data model snapshot".

Reply:
+1 Maybe something like: "When deleting a migration, use the `dotnet ef migrations remove`. This will ensure the snapshot is also correctly reset.